detex.l

```diff
@@ -734,7 +820,9 @@ InputFile(char *sbFile)
             return;
 	} 
 #ifdef FLEX_SCANNER
-        rgsb[csb++] = YY_CURRENT_BUFFER;
+        rgsb[csb++]     = YY_CURRENT_BUFFER;
+        fFileLines[csb] = 1;
+        stracpy(&fFileNames[csb], sbFile);
```

Comment:
This is not correct according to Valgrind: https://travis-ci.org/github/pkubowicz/opendetex/jobs/697636653

Reply:
Good call, forgot to allocate space for `'\0'`